WORKSPACE

```diff
@@ -171,7 +171,7 @@ http_archive(
 git_repository(
     name = "mediapipe",
     remote = "https://github.com/openvinotoolkit/mediapipe",
-    commit = "8a7fff74151dd112d0aae87d8c74fb0d07e206ec", # Replace for windows (#105) 21/01/2025
+    commit = "673490d3e3e9e8f00c21da7ed4f3891338184459", # deprecated-ov branch
```

Comment:
`deprecated-ov` doesn't sound convincing...

Reply:
changed to main since we merged MP commit